pandas/tests/scalar/timestamp/test_unary_ops.py

```diff
@@ -328,6 +328,17 @@ def test_replace_dst_border(self):
         expected = Timestamp('2013-11-3 03:00:00', tz='America/Chicago')
         assert result == expected
 
+    # --------------------------------------------------------------
+    # Timestamp.normalize
+
+    @pytest.mark.parametrize('arg', ['2013-11-30', '2013-11-30 12:00:00'])
```

Comment:
is there a normalize_nat test as well?

Reply:
We could have one for Timstamp mirroring (another issue). Probably would just return `NaT`